src/controllers/exercise/types.ts

```diff
@@ -0,0 +1,17 @@
+export interface FindExerciseRequest {
+  id: string;
+}
+
+export interface CreateExerciseRequest {
+  name?: string;
+  photo?: string;
+  categories: string[];
+  machine_id: string;
```

Comment:
optional on all, since we don't know what the request from the client is.

the verification is done on code, to validate that it does exist.